lib/web/question_router.ts

```diff
@@ -1494,6 +1494,16 @@ async function getQuestionsUserCreatedOrForecastedOnOrIsSharedWith(
                       },
                     },
                   },
+                  {
+                    tags: {
+                      some: {
+                        name: {
```

Comment:
I think we need to check that the tag.userId === this user here

Note that tags are user-specific - you can only see tags that you add to a question. This is because questions might mean different things to different forecasters, e.g. I might tag your question #friend-advice and #quick-predictions and you might tag it #high-effort-predictions

Reply:
Makes sense - I'll update this, thanks

Reply:
I've updated this to add the userId check, see slack